UPGRADE-6.3.md

```diff
@@ -27,6 +27,12 @@ The system now uses Symfony 3.4, which needs a few adjustments:
 The system now uses Twig 2.x. Please have a look at the Twig changelog for required adjustments, but major problems are
 not expected.
 
+## Mailer Peer Security
+
+The default value of config key `chameleon_system_core: mailer: peer_security` was changed from "permissive" to "strict".
+This means that the class `\ChameleonSystem\CoreBundle\Security\Https\StrictHttpsContext` is used for the settings
```

Comment:
Suggestion: "Using this setting, SMTP connections verify SSL/TLS certificate validity so that invalid or self-signed certificates are rejected." as replacement for the second sentence. I think the purely technical information part is not interesting here.